Scripts/Mac/CI/CreateBuildDrop.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+. "$(dirname ${BASH_SOURCE[0]})/../InitializeEnvironment.sh"
+
+CONFIGURATION=$1
+BUILDDROP_ROOT=$2
+if [ -z $BUILDDROP_ROOT ] || [ -z $CONFIGURATION ]; then
+  echo 'ERROR: Usage: CreateBuildDrop.sh [configuration] [build drop root directory]'
+  exit 1
+fi
+
+# Set up some paths
+BUILDDROP_BUILDOUTPUT=$BUILDDROP_ROOT/BuildOutput
+BUILDDROP_SRC=$BUILDDROP_ROOT/src
+BUILDDROP_PROJFS=$BUILDDROP_SRC/ProfFS.Mac
+BUILDDROP_KEXT=$BUILDDROP_BUILDOUTPUT/ProjFS.Mac/Native/Build/Products/$CONFIGURATION
+
+# Set up the build drop directory structure
+rm -rf $BUILDDROP_ROOT
+mkdir -p $BUILDDROP_BUILDOUTPUT
+mkdir -p $BUILDDROP_SRC
+mkdir -p $BUILDDROP_PROJFS
+mkdir -p $BUILDDROP_KEXT
```

Comment:
There are cases in the build scripts that use `sudo`.  How do we know when we will need to use that vs not?

Reply:
I'm making the assumption in this script that the CI agent will have access to wherever it is asked to publish the drop. Also, this caught my clumsiness a few times where I fat fingered /[something] instead of ~/[something].